auth-starter/src/main/java/com/vaadin/auth/starter/VaadinAuthSecurityConfiguration.java

```diff
@@ -0,0 +1,60 @@
+package com.vaadin.auth.starter;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.oauth2.client.OAuth2LoginConfigurer;
+import org.springframework.util.StringUtils;
+
+import com.vaadin.flow.server.auth.ViewAccessChecker;
+import com.vaadin.flow.spring.security.VaadinWebSecurity;
+
+/**
+ * This configuration bean is provided is no other {@link VaadinWebSecurity}
```

Comment:
Shouldn't it be 'if no other'?

Reply:
Yes, it should 😅 Also the JavaDoc is not correct since the bean is not conditional, so I've pushed another commit making the bean conditional on a new `vaadin.auth.auto-configure` property (and also added new tests for that).